Mathlib/Data/Finset/Card.lean

```diff
@@ -26,6 +26,9 @@ This defines the cardinality of a `Finset` and provides induction principles for
 * `Finset.Nonempty.strong_induction`
 -/
 
+-- TODO
+-- assert_not_exists OrderedCommMonoid
+-- assert_not_exists Ring
```

Comment:
FYI, this TODO is waiting on #12836

Reply:
Can you fix it now?

Reply:
Actually no I can't. I was wrong. Removing `Ring` depends on #12845 instead. Removing `OrderedCommMonoid` seems very hard right now.